tools/IceRpc.BuildTelemetry.Reporter/Program.cs

```
@@ -2,98 +2,92 @@
-        .Skip(1)
-        .FirstOrDefault(), out var result) && result;
-
-    // Parse the contains-slice2 argument
-    bool containsSlice2 = bool.TryParse(args
-        .SkipWhile(arg => arg != "--contains-slice2")
-        .Skip(1)
-        .FirstOrDefault(), out result) && result;
-
-    // Parse the src-file-count argument
-    int sourceFileCount = int.TryParse(args
-        .SkipWhile(arg => arg != "--src-file-count")
-        .Skip(1)
-        .FirstOrDefault(), out var sourceFileCountResult) ? sourceFileCountResult : 0;
-
-    // Parse the ref-file-count argument
-    int referenceFileCount = int.TryParse(args
-        .SkipWhile(arg => arg != "--ref-file-count")
-        .Skip(1)
-        .FirstOrDefault(), out var referenceFileCountResult) ? referenceFileCountResult : 0;
-
-    BuildTelemetry buildTelemetry = idl.ToLower() switch
-    {
-        "slice" => new BuildTelemetry.Slice(new SliceTelemetryData(
-            version,
-            SystemHelpers.GetOperatingSystem(),
-            SystemHelpers.GetArchitecture(),
-            SystemHelpers.IsCi(),
-            new TargetLanguage.CSharp(Environment.Version.ToString()),
-            compilationHash,
-            containsSlice1,
-            containsSlice2,
-            sourceFileCount,
-            referenceFileCount)),
-        "protobuf" => new BuildTelemetry.Protobuf(new ProtobufTelemetryData(
-            version,
-            SystemHelpers.GetOperatingSystem(),
-            SystemHelpers.GetArchitecture(),
-            SystemHelpers.IsCi(),
-            new TargetLanguage.CSharp(Environment.Version.ToString()),
-            sourceFileCount,
-            compilationHash)),
-        _ => throw new ArgumentException($"Unknown IDL: {idl}") // This should never happen
-    };
-
-    // Upload the telemetry to the server.
-    await reporter.UploadAsync(buildTelemetry, cancellationToken: cts.Token);
-
-    // Shutdown the connection.
-    await connection.ShutdownAsync(cts.Token);
-}
-catch
```

Just removed this catch block that was ignoring all exceptions